flink-next-gen-connector-influxdb/src/main/java/org/apache/flink/streaming/connectors/influxdb/source/reader/InfluxDBSplitReader.java

```diff
@@ -209,6 +210,25 @@ private void sendResponse(
         }
     }
 
+    private class InfluxDBHealthCheckHandler implements HttpHandler {
+        @Override
+        public void handle(final HttpExchange t) throws IOException {
+            sendResponse(t, 200, "ready for writes");
+        }
+
+        private void sendResponse(
```

Comment:
That code is duplicated from the other handler. Any good ideas where to put it? 🤔

Reply:
You could implement an abstract class InfluxDBAPIHandler that implements that method and than inherit from this class ...